### PR TITLE
Update Tea Time scheduling

### PR DESCRIPTION
**Pull request by ankitshah811**

### CODE REVIEW

File: app/views/shared/_tea_times.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      <!--TODO: Remove after Forbes/Inc-->
+      <%= 'Join for an upcoming tea time with #{host.name}' %>
+    </h3>
+  <% end %>
+  <div class="tt-set-container">
+    <% tea_times.each do |tt| %>
+      <%= render partial: 'shared/tea_time_row', locals: {tea_time: tt} %>
+    <% end %>
+  </div>
+  <% if (host ||= nil) %>
```
Comment (mehulkar) [inline app/views/shared/_tea_times.html.erb]: @ankitshah811 what were you trying to do here? Is it supposed to be `if not a host` ?

File: app/views/cities/forbes_show.html.erb [hunk 2]
```diff
@@ -25,7 +25,8 @@
   	<% elsif @city.cold_water? %>
   	  <%= render partial: 'cities/show/cold_water' %>
   	<% else %>
-  	  <%= render partial: 'cities/show/active_waitlisted' %>
+  	  <!-- <%= render partial: 'cities/show/active_waitlisted' %> -->
+      <%= render partial: 'cities/show/active' %>
```
Comment (ankitshah811) [inline app/views/cities/forbes_show.html.erb]: This *requires* that new users stop being waitlisted, as the site acts funny if a user who's on the waitlist can see tea times and signs up for them. They end of getting signed up, but the site tells them they can't do it (even though they...actually get signed up)

However, if a new user goes to a pending city page or creates a new city, the flow on the new site should not change. They should still see `cold_water` and `unconfirmed` for cities that fall under those categories.

Comment (mehulkar) [inline app/views/cities/forbes_show.html.erb]: hmm, not entirely sure what this is, but opened to #431 to circle back around

File: app/views/tea_time_mailer/followup_no_show.markerb [hunk 3]
```diff
@@ -1,28 +1,47 @@
...
+text you a reminder? Or should we have a policy that penalizes no shows 
+somehow? This is my actual email, so you can reply and let me know. 
+
+Also, [sign up for tea time again.](<%=url_for(controller: 'cities', 
+															action: 'show', 
+															id: @tea_time.city.city_code, 
+															only_path: false, 
+															utm_campaign: 'followupnoshow', 
+															utm_medium: 'email', 
+															utm_content: 'show') %>)
```
Comment (ankitshah811) [inline app/views/tea_time_mailer/followup_no_show.markerb]: LOL these were not supposed to be so spaced out

File: app/views/shared/_tea_times.html.erb (hunk 1 above)
Comment (ankitshah811) [inline app/views/shared/_tea_times.html.erb]: This is actually not something I wrote. This was just converted from the haml version. It was a dumb conditional put in place to accommodate the tea time partial that was placed at the bottom of host profiles

File: app/views/cities/forbes_show.html.erb (hunk 2 above)
Comment (ankitshah811) [inline app/views/cities/forbes_show.html.erb]: if confused, let's phone after getting other stuff sorted tonight

**Pull request merged**